specs/commandline/sign.md

```diff
@@ -37,8 +37,44 @@ Flags:
       --plugin-config strings    {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
   -u, --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
+      --use-image-manifest       optional for using OCI image manifest to store signatures in a registry
 ```
 
+## Use OCI image manifest to store signatures
+
+By default, Notation uses [OCI artifact manifest][oci-image-manifest] to store signatures in the registries. For backward compatibility, Notation supports using `OCI image manifest` to store the signature in the registries that implement partial or older versions of the OCI Image specification. Use flag `--use-image-manifest` to force Notation store the signatures using OCI image manifest. Note that Notation doesn't automatically switch to `OCI image manifest` for a failure of using `OCI artifact manifest`.
```

Comment:
> that implement partial or older versions of the OCI Image specification

Older than what version? Can we please add more information here?

I would remove ```Note that Notation doesn't automatically switch to `OCI image manifest` for a failure of using `OCI artifact manifest``` because its implicit from explicitly using a flag. Also, when we do automatic fallback, we will need to delete it and which we might forget.